template-for-proof/Makefile

```diff
@@ -4,6 +4,10 @@
 HARNESS_ENTRY=harness
 HARNESS_FILE=<__FUNCTION_NAME__>_harness
 
+# This should be a unique identifier for this proof, and will appear on the
+# Litani dashboard. It can be human-readable and contain spaces if you wish.
+PROOF_ID = <...>
```

Comment:
This needs to have a default value, can we use <__FUNCTION_NAME__> for the default?

Can we use a name like PROOF_UID that emphasizes the need for a UID and not just an ID?

Reply:
Done